tee-worker/litentry/core/assertion-build/src/dynamic/contracts/token_holding_amount/ERC20.sol

```diff
@@ -17,34 +17,40 @@
 // SPDX-License-Identifier: GPL-3.0-or-later
 
 pragma solidity ^0.8.8;
+
 import "../libraries/Identities.sol";
 import "../libraries/Utils.sol";
 import { TokenHoldingAmount } from "./TokenHoldingAmount.sol";
 import { NoderealClient } from "./NoderealClient.sol";
-abstract contract ERC20 is TokenHoldingAmount {
-	mapping(uint32 => string) internal networkTokenAddresses;
+import { Ada } from "./erc20/Ada.sol";
+import { Amp } from "./erc20/Amp.sol";
```

Comment:
similar problem as above. 
Do I misunderstand anything?

Reply:
I added all and made some changes to the positions.